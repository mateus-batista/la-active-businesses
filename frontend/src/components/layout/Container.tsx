import styled from "@emotion/styled";

export const Container = styled.div`
  display: grid;
  justify-content: stretch;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-template-rows: 50px calc(100vh - 100px) 50px;
  grid-template-areas:
    "header header header header"
    "main main main main"
    "footer footer footer footer";
`;
