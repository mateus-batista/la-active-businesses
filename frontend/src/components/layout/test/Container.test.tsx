import React from "react";
import { render } from "@testing-library/react";
import { Container } from "../Container";

it("should render correctly", () => {
  expect(render(<Container />)).toMatchSnapshot();
});
