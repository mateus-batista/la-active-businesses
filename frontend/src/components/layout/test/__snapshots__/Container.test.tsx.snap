// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  display: grid;
  -webkit-box-pack: stretch;
  -ms-flex-pack: stretch;
  -webkit-justify-content: stretch;
  justify-content: stretch;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-template-rows: 50px calc(100vh - 100px) 50px;
  grid-template-areas: "header header header header" "main main main main" "footer footer footer footer";
}

<body>
    <div>
      <div
        class="emotion-0"
      />
    </div>
  </body>,
  "container": .emotion-0 {
  display: grid;
  -webkit-box-pack: stretch;
  -ms-flex-pack: stretch;
  -webkit-justify-content: stretch;
  justify-content: stretch;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-template-rows: 50px calc(100vh - 100px) 50px;
  grid-template-areas: "header header header header" "main main main main" "footer footer footer footer";
}

<div>
    <div
      class="emotion-0"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
